### DISCUSSION TITLE
Clickable anchor links next to headlines

### DISCUSSION THREAD

**Discussion by haesleinhuepf** [chanzuckerberg/napari-hub#345] [Ideas]
Hi all,

I'd like to ask for a feature in the napari-hub plugin pages. It would be great if little anchor icons could be shown (on mouse hover for example) next to headlines in the plugin pages. Me, a plugin developer, could then right-click the anchor and easily get a link to the installation instrucitons of a plugin for example and send it to a potential user. On github.com this is implemented for readme files for example:

![image](https://user-images.githubusercontent.com/12660498/143735094-f5e5b2c7-ad12-490a-a8b3-7140dbc96473.png)

Let me know what you think. Thanks!

Best,
Robert

**Comment by haesleinhuepf** [comment]
Note: The anchors are there, they work. It's just that the little icon is missing to retrieve them...

**Comment by neuromusic** [comment]
on it: 
- issue: https://github.com/chanzuckerberg/napari-hub/issues/478 
- PR: https://github.com/chanzuckerberg/napari-hub/pull/489

**Comment by haesleinhuepf** [comment]
Hi @neuromusic ,

just a minor follow-up idea: Would it be possible to have the anchor also next to headlines of level H2 and H3? At the moment, the anchors only show up on level H1:

![image](https://user-images.githubusercontent.com/12660498/170518914-929a5aa1-6809-456e-8165-61375e947506.png)

Thanks!
Robert

**Reply by codemonkey800** [reply]
ahh sounds good, not sure how I missed this 😆 An issue has been created and will be worked on soon:

https://github.com/chanzuckerberg/napari-hub/issues/504
